python/src/scipp/compat/mantid.py

```diff
@@ -768,6 +768,9 @@ def load(filename="",
     :param str mantid_alg: Mantid algorithm to use for loading. Default is
                            `Load`.
     :param dict mantid_args: Dict of keyword arguments to forward to Mantid.
+    :param bool advanced_geometry: If True, load the full detector geometry
+                                   including shapes and rotations. If False,
+                                   load only the detector position.
```

Comment:
This actually also affects there spherical averaging for detector groups, doesn't it?